cl/recap/mergers.py

```diff
@@ -834,7 +834,17 @@ async def add_docket_entries(
                 "Multiple recap documents found for document entry number'%s' "
                 "while processing '%s'" % (docket_entry["document_number"], d)
             )
-            continue
+            if params["document_type"] == RECAPDocument.ATTACHMENT:
+                continue
+            duplicate_rd_queryset = RECAPDocument.objects.filter(**params)
+            rd_with_pdf_queryset = duplicate_rd_queryset.filter(
+                is_available=True
+            ).exclude(filepath_local="")
+            if await rd_with_pdf_queryset.aexists():
+                rd = rd_with_pdf_queryset.alatest("date_created")
```

Comment:
Oh, I think these comments remained in draft, sorry.

It seems here it needs `await` otherwise it'll trigger an error:
```suggestion
                rd = await rd_with_pdf_queryset.alatest("date_created")
```

Reply:
fixed